src/main/java/assets/css/AcesUp.css

```diff
@@ -1,65 +1,113 @@
+body{
+    background-image: url("../images/background.jpg");
+    background-size: 100%;
```

Comment:
Yea, it's a little confusing as written. As I understand it, the background size specifies the the size the image itself (in this case the png we're using). The height property serves a different purpose here. The reason I put the height in there is so that I could control the sizing of the divs within the body.

I could be wrong, and I did not try the page after removing either of these properties, but i'll give it a shot if we have another iteration of this assignment